Automattic/wp-calypso
# Signup: Adding site style to the business site segment flow

## Changes proposed in this Pull Request

Break out the 🍾 This PR adds the **site style step** to the business onboarding flow.

~~⚠️ #32135 should be merged first.~~ ✅ 

![Apr-23-2019 12-40-00](https://user-images.githubusercontent.com/6458278/56549025-00545400-65c5-11e9-8622-b75612b85399.gif)

## Testing instructions

1. With **Business** as your site segment, continue through the flow to http://calypso.localhost:3000/start/onboarding-for-business/site-style-with-preview
2. Select a site style. The site preview should update according to the selected style.
3. Check the step window in various screen widths. The buttons should resize accordingly.
4. Create a site. Your chosen site style should determine the site's ultimate theme.
